libyul/YulStack.cpp

```diff
@@ -396,9 +396,10 @@ Json YulStack::cfgJson() const
 	auto exportCFGFromObject = [&](Object const& _object) -> Json {
 		// NOTE: The block Ids are reset for each object
 		std::unique_ptr<ControlFlow> controlFlow = SSAControlFlowGraphBuilder::build(
-			*_object.analysisInfo.get(),
+			*_object.analysisInfo,
 			languageToDialect(m_language, m_evmVersion, m_eofVersion),
-			_object.code()->root()
+			_object.code()->root(),
+			true
```

Comment:
We're now defaulting to leaving variable assignments untouched then? Can you add a comment next to the `true` to indicate what it does?

Reply:
Yep, done!